nixos/tests/all-tests.nix

```diff
@@ -620,6 +620,7 @@ in {
   systemd-misc = handleTest ./systemd-misc.nix {};
   tandoor-recipes = handleTest ./tandoor-recipes.nix {};
   taskserver = handleTest ./taskserver.nix {};
+  tayga = handleTest ./tayga.nix {};
```

Comment:
Better to test both the legacy and networkd implementation.

Reply:
Inside the test is done with both the legacy and systemd based network config. That is a small loop in line 251 that tests the routers after each other.